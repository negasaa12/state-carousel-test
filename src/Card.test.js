import React from "react";
import { render, fireEvent } from "@testing-library/react";

import Card from "./Card";






it("should render Card component", ()=>{

    render(<Card/>)
  });
  
  it("matches snapshot of Card component", () =>{
  
    const {asFragment}= render(<Card/>);
    expect(asFragment).toMatchSnapshot();
    
  });
  