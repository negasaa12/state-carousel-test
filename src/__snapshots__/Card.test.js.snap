// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot of Card component 1`] = `[Function]`;
